typing/cmi_format.ml

```diff
@@ -20,8 +20,7 @@ type pers_flags =
   | Unsafe_string
 
 type error =
-    Not_an_interface of string
-  | Wrong_version_interface of string * string
+    Not_an_interface of string * string option
```

Comment:
I'm not sure which exception constructors your are thinking of, but my experience is the opposite: trying to keep the most structured error possible until the printer that is registered by `Location.register...`. See `Includemod` for example. I can't even think of an error type with error messages pre-expanded.
So, for once, the common practice is also the good one. :)